plinth/setup.py

```diff
@@ -158,3 +158,12 @@ def setup_all_modules(essential=False):
             continue
 
         module.setup_helper.run()
+
+def setup_selected(module_list):
+    """Runs setup for selected modules and exit."""
+    logger.info('Running setup for Given modules')
+    for module_name, module in plinth.module_loader.loaded_modules.items():
+        if module_name not in module_list:
+            continue
+
+        module.setup_helper.run()
```

Comment:
Would you please consider merging the two setup methods?  We currently have one line to actually run the setup but that might get expanded later.
Something like
```
def setup_modules(modules_list=None, essential=False):
```